ci/benchmark/build.sh

```diff
@@ -63,10 +63,28 @@ nvidia-smi
 logger "Activate conda env..."
 source activate rapids
 
+
+# Enter dependencies to be shown in ASV tooltips.
+# These dependencies will also be installed by conda.
+# A dependency can exist in both arrays without causing issues.
+CUGRAPH_DEPS=(cudf rmm)
+LIBCUGRAPH_DEPS=(cudf rmm)
+
+# Concatenate dependency arrays, convert to JSON array,
+# and remove duplicates.
+X=("${CUGRAPH_DEPS[@]}" "${LIBCUGRAPH_DEPS[@]}")
+DEPS=$(printf '%s\n' "${X[@]}" | jq -R . | jq -s 'unique')
+
+# Create install args for conda (i.e. "cudf=0.15" "rmm=0.15")
+CONDA_INSTALL=
+for DEP in $(echo "${DEPS}" | jq -r '.[]'); do
+  CONDA_INSTALL+="${DEP}=${MINOR_VERSION}"
+  CONDA_INSTALL+=" "
+done
+
 logger "conda install required packages"
 conda install -c nvidia -c rapidsai -c rapidsai-nightly -c conda-forge -c defaults \
-      "cudf=${MINOR_VERSION}" \
-      "rmm=${MINOR_VERSION}" \
+      ${CONDA_INSTALL} \
```

Comment:
I would revert these lines back. It is important that the conda install lines are easily readable/changeable for the devs.